pkgs/top-level/all-packages.nix

```diff
@@ -29420,11 +29420,11 @@ with pkgs;
 
   scli = callPackage ../applications/misc/scli { };
 
-  scribus = callPackage ../applications/office/scribus {
+  scribus_1_4 = callPackage ../applications/office/scribus/1_4.nix {
     inherit (gnome2) libart_lgpl;
   };
 
-  scribusUnstable = libsForQt5.callPackage ../applications/office/scribus/unstable.nix { };
+  scribus = libsForQt5.callPackage ../applications/office/scribus/default.nix { };
```

Comment:
Couldn't we then name this `scribus_1_5` directly and make `scribus` an alias to that? This way we won't need to shuffle everything around when 1.6 gets released.

Reply:
I'm of a mind to actually delete the older `scribus` versions completely. But that doesn't have to happen right now.

Would you put the alias in `pkgs/top-level/aliases.nix` or right here?

Reply:
Considering that https://www.scribus.net/downloads/ still advertises 1.4 as the stable version I'd rather keep it as long as that doesn't change. I think we just can add `scribus = scribus_1_5;` here in that file.
Also it might make sense to mention the change in the release notes, so users that are depending on 1.4 get warned that they need to reference it explicitly. 1.4 and 1.5 have quite different user interfaces as far as I remember.

Reply:
> https://www.scribus.net/downloads/ still advertises 1.4 as the stable version

Thanks, I didn't notice! Will do.

Reply:
scribus 1.4 depends on python2 and qt4, which have both been unsupported for years. I'm in favor of getting rid of it, even if the download page says that this is the stable version.

Reply:
@ajs124 Makes sense to me, but let's discuss that in another issue.